src/App/Range.cpp

```diff
@@ -234,24 +234,28 @@ App::CellAddress App::stringToAddress(const char * strAddress, bool silent)
   * @returns Address given as a string.
   */
 
-std::string App::CellAddress::toString(bool noAbsolute) const
+std::string App::CellAddress::toString(bool noAbsolute, bool r, bool c) const
```

Comment:
@chennes 
This is a code smell called _boolean blindness_

For a reviewer or programmer it's not obvious what the arguments are supposed to do and one has to go to the declaration or documentation. Since it's not even documented one has to dive into the code to get a rough idea.

Therefore I suggest to replace the three boolean arguments with an enum class that supports bit-wise OR. Then in client code some speaking identifiers must be used that increases readability a lot.

Reply:
See https://github.com/FreeCAD/FreeCAD/pull/5360

Reply:
That looks excellent, thanks for adding that enum bitmask metaprogramming.

Reply:
It's basically the same as Qt's QFlags class but I didn't want to pull-in a Qt dependency on this class. And the bitmask is something that we may need more often for non-Qt classes.

If you have a minute you can try to compile it on Windows, otherwise I will do it the next time I am on Windows. If MSVC accepts it the PR can be merged.

Reply:
Compiling now...